src/index.ts

```diff
@@ -63,6 +63,12 @@ const submitFeedbackForPR = async (
       let notes: string;
       if (splitNotes.length > 0) {
         notes = splitNotes.map(line => `> ${line}`).join('\n');
+
+        // remove any markdown comments
+        const commentMatch = /<!--.*-->/g.exec(notes);
```

Comment:
We don't want to infinitely match, we want to match the shortest possible `.*`

```suggestion
        const commentMatch = /<!--.*?-->/g.exec(notes);
```

Reply:
`?` added in 9dd052fed525417e1156b87c24a321c063d6b799